decisions/0010-sunsetting-sandbox.md

```diff
@@ -0,0 +1,75 @@
+---
+status: accepted
+date: 2024-08-14
+deciders: Montana Smith, NMDC PIs, Mike Nagler, Patrick Kalita 
+consulted: Entire NMDC team via [GitHub discussion](https://github.com/microbiomedata/issues/discussions/743), meetings, and slack were given the oppotrunity to voice opinions
+---
+# Sunsetting the NMDC Sandbox UI
+
+This ADR outlines the process we used to decide to sunset the NMDC Sandbox Submission Portal, the decision that was made, and the planned updated to ensure functionality is maintained. 
+
+## Context and Problem Statement
+
+Currently, the NMDC submission portal sandbox is wiped every Sunday.
+Some ambassadors or other users are still using the production environment for tests, workshops, and fake submissions. Additionally, people don't always know about the NMDC Sandbox site or there's a desire to have fake/test studies persist after a Sunday (ex: Prep for a workshop on Friday, and the submission still be there for Monday's session; Re-use the same submission for multiple workshops & not need to re-submit)
+
+We proposed a few solutions in the [GitHub dsicussion](https://github.com/microbiomedata/issues/discussions/743) and in a small meeting (contact Montana for those google notes). Team members were invited to contribute to and voice opinions on the GitHub discussion. Additionaly, we polled slack for reactions to confirm people were aware and we received 12 approvals. 
+
+<!-- This is an optional element. Feel free to remove. -->
+## Decision Drivers
+
+* People aren't using the sandbox site 
```

Comment:
I assume some people have been using the sandbox environment (e.g. during workshops). 🤷 

I changed this to emphasize that the sandbox environment isn't housing _all_ test submissions.

```suggestion
* Even though we have been maintaining a sandbox environment, many people have continued to create test submissions in the production environment.
```